### PR TITLE
Add sections about controlling camera, animations and audio

### PR DESCRIPTION
**Pull request by graouts**

### CODE REVIEW

File: model/README.md [hunk 1]
```diff
@@ -196,6 +199,76 @@ model in augmented reality to allow the user to visualize it at real scale in th
 surroundings. To support this, new DOM APIs may be added or the existing HTML Fullscreen API extended
 via more [FullscreenOptions](https://fullscreen.spec.whatwg.org/#dictdef-fullscreenoptions) properties.
 
+### Controlling the camera
+
+While by setting the `interactive` property the author will allow a built-in behavior such that dragging
```
Comment (smfr) [inline model/README.md:204]: This sentence is hard to parse, because it starts with "While by"

File: model/README.md [hunk 2]
```diff
@@ -196,6 +199,76 @@ model in augmented reality to allow the user to visualize it at real scale in th
...
+over a `<model>` element would result in modifying the camera, we propose to allow authors direct control
+of the camera via DOM APIs. An initial proposition would be to add an `HTMLModelElementCamera`:
+
+```idl
+dictionary HTMLModelElementCamera {
```
Comment (smfr) [inline model/README.md:209]: But you can't change the camera location (affection foreshortening)? Calling this "the camera" might be a little shortsighted; it's camera coordinates.

File: model/README.md [hunk 3]
```diff
@@ -196,6 +199,76 @@ model in augmented reality to allow the user to visualize it at real scale in th
...
+    double scale;
+};
+```
+
+Then the camera can be set and read back:
+
+```idl
+interface HTMLModelElement : HTMLElement {
+    Promise<HTMLModelElementCamera> getCamera();
+    Promise<undefined> setCamera(HTMLModelElementCamera camera);
```
Comment (smfr) [inline model/README.md:221]: Really setting a viewpoint.

File: model/README.md [hunk 4]
```diff
@@ -196,6 +199,76 @@ model in augmented reality to allow the user to visualize it at real scale in th
...
+}
+```
+
+Note the use of promises since it is likely that the model is rendered out-of-process and any communication
+with that process would need to be asynchronous. This applies to other promise-based APIs discussed in this
+document.
+
+### Controlling animations
+
+Formats supported by `<model>` may support animations built into the resource itself, such as the  USDZ
```
Comment (smfr) [inline model/README.md:231]: double space

File: model/README.md [hunk 5]
```diff
@@ -196,6 +199,76 @@ model in augmented reality to allow the user to visualize it at real scale in th
...
+important topic would be whether the Web Animations specification could be leveraged to expose and control
+animations for the resource.
+
+For experimental purposes, we propose an initial, basic set of DOM APIs based on the assumption that
+a single animation is controlled. With this proposal the author could control whether the animation is
+playing, looping, query its duration and set its current time, allowing the creation of controls to
+toggle playback and scrub through the animation.
+
+```idl
+interface HTMLModelElement : HTMLElement {
```
Comment (smfr) [inline model/README.md:244]: Might be nice to say how this fits with Web Animation timelines. Is this itself a timeline, or does it provide or subscribe to one?

File: model/README.md [hunk 6]
```diff
@@ -196,6 +199,76 @@ model in augmented reality to allow the user to visualize it at real scale in th
...
+
+    Promise<double> animationDuration();
+    Promise<double> animationCurrentTime();
+    Promise<undefined> setAnimationCurrentTime(double currentTime);
+}
+```
+
+### Controlling audio
+
+Another feature that may be supported by the resource file format is audio. Much like animations,
```
Comment (smfr) [inline model/README.md:260]: Does audio start/stop when animation is played/paused? Unclear if audio muted and playback rate are tied to other states of the model.

File: model/README.md (hunk 3 above)
Comment (grorg) [reply model/README.md:221]: 3d formats tend to call these cameras even if it doesn't make complete sense here.  (they might also have different properties, like fov/lens)

**Review by grorg — APPROVED** [review]
